server/src/services/library.service.ts

```diff
@@ -521,6 +529,8 @@ export class LibraryService extends EventEmitter {
       await this.assetRepository.updateAll([existingAssetEntity.id], {
         fileCreatedAt: stats.mtime,
         fileModifiedAt: stats.mtime,
+        localDateTime: stats.mtime,
```

Comment:
Does this need to be offset so it's in utc?

Reply:
hm...good point. Maybe let's not touch that rabbit hole today